pkg/agent/watcher_test.go

```diff
@@ -82,7 +83,19 @@ var _ = Describe("Watcher", func() {
 						},
 					},
 				}
-				watcher.parseConfig(modelConfigs, false)
+				_, err := os.Stat("/tmp/configs")
```

Comment:
was there any issue with the original test case?

Reply:
There were lines of code that were not covered with the original one.